frontend/src/queries/nodes/DataTable/dataTableLogic.ts

```diff
@@ -162,8 +162,7 @@ export const dataTableLogic = kea<dataTableLogicType>([
                         showReload: query.showReload ?? showIfFull,
                         showTimings: query.showTimings ?? flagQueryTimingsEnabled,
                         showElapsedTime:
-                            query.showTimings ||
-                            flagQueryTimingsEnabled ||
+                            (query.showTimings ?? flagQueryTimingsEnabled) ||
```

Comment:
@mariusandra I'm assuming this was your intention here, as otherwise I can't override things